cilium/cmd/monitor.go

```diff
@@ -287,7 +287,9 @@ start:
 			}
 		}
 
-		if pl.Type == payload.EventSample {
+		if meta.Size == 0 {
```

Comment:
When there are no monitors, is that what produces an API response with size 0 here?

Reply:
The intention of writing an empty meta is so that we can detect a write failure when any of the known connections are gone / inactive. If we are not aware of any connections we never write anything and these lines are not reached. The connections are added to a list of listeners, but the Go API has no good way for us to know when a connection is inactive. So we need to try to write in order to detect a failure. Does this answer your question?

Reply:
The explanation makes sense, can you add it as a comment to the code? It's not obvious otherwise.

Reply:
sure